docs/code/code-install-lang-server.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[code-install-lang-server]]
+== Install language server
+
+Code builds in support for several languages. You can install additional languages as a {kib} plugin.
+
+[role="screenshot"]
+image::images/code-lang-server-tab.png[]
+
+The following languages are supported for the current version:
+
+Built-in language support:
```

Comment:
I recommend making this a bulleted list:

- Built-in language support:  `Typescript`
- Additional language support:  `Java`

Also, I think we just need to put Typescript and Java in code font, with no punctuation around it.